src/main/frontend/src/launcher-quarkus/pickers/extensions-picker.tsx

```diff
@@ -101,40 +101,25 @@ function Extension(props: ExtensionProps) {
   )
 }
 
-export const filterFunction = (filter: string) => (d: ExtensionEntry) => {
-  const filterLowerCase = filter.trim().toLowerCase();
-  if (!filterLowerCase) {
-    return true;
-  }
-  const shortName = d.shortName ? d.shortName.toLowerCase() : '';
-  if (filterLowerCase === shortName) {
-    return true;
-  }
-  return d.name.toLowerCase().includes(filterLowerCase)
-    || d.labels.filter(l => l.startsWith(filterLowerCase)).length > 0
-    || (d.category && d.category.toLowerCase().startsWith(filterLowerCase))
-    || shortName.startsWith(filterLowerCase);
-}
-
-export const sortFunction = (filter: string) => (a: ExtensionEntry, b: ExtensionEntry) => {
-  const filterLowerCase = filter.trim().toLowerCase();
-  if (!filterLowerCase) {
-    return a.order > b.order ? 1 : -1;
-  }
-  const startWithAShortName = !!a.shortName && a.shortName.toLowerCase().startsWith(filterLowerCase);
-  const startWithBShortName = !!b.shortName && b.shortName.toLowerCase().startsWith(filterLowerCase);
-  if (startWithAShortName !== startWithBShortName) {
-    return startWithAShortName ? -1 : 1;
-  }
-  const startWithOneOfALabel = a.labels.filter(l => l.startsWith(filterLowerCase)).length > 0;
-  const startWithOneOfBLabel = b.labels.filter(l => l.startsWith(filterLowerCase)).length > 0;
-  if (startWithOneOfALabel !== startWithOneOfBLabel) {
-    return startWithOneOfALabel ? -1 : 1;
-  }
-  if (a.name.toLowerCase().startsWith(filterLowerCase) !== b.name.toLowerCase().startsWith(filterLowerCase)) {
-    return a.name.toLowerCase().startsWith(filterLowerCase) ? -1 : 1;
-  }
-  return a.order > b.order ? 1 : -1;
+function Blurb() {
+  const [visible, setVisible] = useSessionStorageWithObject<Boolean>('quarkus-blurb-visible', true);
+  return (
+    <>
+      {visible && (
+        <div className="quarkus-blurb">
+          <div className="blurb-icon"><InfoIcon /></div>
+          <div className="blurb-content">
+            <p>This page will help you bootstrap your Quarkus application and discover its extensions ecosystem.</p>
+            <br />
+            <p>Think of Quarkus extensions as your project dependencies. Extensions configure, boot and integrate a framework or technology into your Quarkus application. They also do all of the heavy lifting of providing the right information to GraalVM for your application to compile natively.</p>
+            <br />
+            <p>Explore the wide breath of technologies Quarkus applications and generate your project!</p>
```

Comment:
I think it should be `wide breadth`
`of technologies Quarkus applications` doesn't sound right, looks like there's some words missing